trillian/migrillian/README.md

```diff
@@ -0,0 +1,19 @@
+Migrillian Tool
```

Comment:
Can we add text that describes the status of the tool. Is it beta for example? It looks like it's not quite ready for production yet but it helps users to be clear about it.

Reply:
Done.